requirements/base.in

```diff
@@ -17,4 +17,4 @@ pony
 psycopg2==2.9.10
 psycopg2-binary==2.9.10
 
--e git+https://github.com/gridsingularity/gsy-framework@master#egg=gsy_framework
+-e git+https://github.com/gridsingularity/gsy-framework@feature/GSYE-498#egg=gsy_framework
```

Comment:
I think that this needs to be reverted

Reply:
Hmm, I thought I had fixed that. Will do now